lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -316,8 +316,10 @@ def digital_wallet_payment_method?(payment_method)
         payment_method.source == :google_pay || payment_method.source == :apple_pay
       end
 
-      def adding_network_token_card_data?(payment_method)
-        return true if payment_method.is_a?(ActiveMerchant::Billing::NetworkTokenizationCreditCard) && payment_method.source == :network_token
+      def adding_network_token_card_data?(payment_method, options = {})
+        return true if payment_method.is_a?(ActiveMerchant::Billing::NetworkTokenizationCreditCard) &&
```

Comment:
Can we just leave this method as it was before? I rather remove it from `add_network_token_cryptogram_and_eci`. `adding_network_token_card_data?` is used in other methods and I wouldn't want us to break anything else.

Reply:
I agree, done!